src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -1,13 +1,25 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="ShimReferencePathsWhenCommonTargetsDoesNotUnderstandReferenceAssemblies"
+          BeforeTargets="CoreCompile"
+          Condition="'@(ReferencePathWithRefAssemblies)' == ''">
+    <!-- Common targets should populate this item from dev15.3, but this file
+         may be used (via NuGet package) on earlier MSBuilds. If the
+         adjusted-for-reference-assemblies item is not populated, just use
+         the older item's contents. -->
+    <ItemGroup>
```

Comment:
The new common targets from Microsoft/msbuild#2039 unconditionally populate `@(ReferencePathWithRefAssemblies)` [here](https://github.com/Microsoft/msbuild/pull/2039/files#diff-02872751bc64e0ff785c9b6c1cd919b5R2033), so no--it shouldn't be empty unless `@(ReferencePath)` was too, in which case this should be harmless.